### DISCUSSION TITLE
[A-L] (2023/24) Foglio 8 - Esercizio 4

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#319] [Esercizi - Viaggi/Piazza]
![Screenshot 2023-11-17 184704](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/1b76e4a0-e5c3-44c7-92cc-8aaef09b49e5)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/9a6de931-e827-429f-890d-6e369872e4b5)

**Comment by daromacarlo** [comment]
![Immagine 2024-01-06 191353](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/36f4213b-f766-40a8-8264-63462df1b1c7)
![Immagine 2024-01-06 191409](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/e4ad8044-bda2-432d-9b72-4ede577314ee)
![Immagine 2024-01-06 191421](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/3d6b3d3e-707f-4e62-9ff8-b5471a9fd8d9)

![Immagine 2024-01-06 191337](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/34b31bf2-3f3e-42c4-a32a-8b6fe33d1ef7)
